FrEee.UI.WinForms/Forms/EmpireSetupForm.cs

```diff
@@ -204,18 +204,18 @@ private void btnCompareCultures_Click(object sender, EventArgs e)
 		this.ShowChildForm(new CultureComparisonForm());
 	}
 
-	private void btnLoadRace_Click(object sender, EventArgs e)
-	{
-		var dlg = new OpenFileDialog();
-		dlg.InitialDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "Races");
-		dlg.Filter = "Races (*.rac)|*.rac";
-		var result = dlg.ShowDialog();
-		if (result == DialogResult.OK)
-		{
-			EmpireTemplate.PrimaryRace = Race.Load(dlg.FileName);
-			Bind();
-		}
-	}
+	//private void btnLoadRace_Click(object sender, EventArgs e)
+	//{
+	//	var dlg = new OpenFileDialog();
+	//	dlg.InitialDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "Races");
+	//	dlg.Filter = "Races (*.rac)|*.rac";
+	//	var result = dlg.ShowDialog();
+	//	if (result == DialogResult.OK)
+	//	{
+	//		EmpireTemplate.PrimaryRace = Race.Load(dlg.FileName);
+	//		Bind();
+	//	}
+	//}

```

Comment:
Might as well delete this function if it's no longer being used.

Reply:
I've deleted the two functions in the new commit just now.